### PR TITLE
add SovereignSignal  delegate 

### PR DESCRIPTION
**Pull request by SovereignSignal**
adding sov

# Please Read and check all that apply 

## general 

- [x] no lint or test errors

## If adding delegate

In order to prevent impersonation attacks we ask that the entity to be added as a delegate make a public post either with a link to this pull request or including the same address referenced in the pull request. This post should be on a platform you have already established reputation on. So if you are active on Celo Forum post there or if you have an active Twitter account post there. 

- LINK TO PROOF 

Checklist of common mistakes

- [x] the image is located in the public/logos directory
- [ ] the image filename has zero spaces in it
- [x] the image path in the json exactly matches the actual file name

### CODE REVIEW

**Comment by SovereignSignal** [comment]
Adding a tweet here to confirm my interest: https://x.com/sovereignsignal/status/1830070512581820628

File: src/config/delegates.json [hunk 1]
```diff
@@ -10,6 +10,18 @@
     },
     "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
     "description": "Stake Capital is a trusted and experienced blockchain investment firm, actively contributing to the blockchain ecosystem since 2016. With expertise in venture capital, incubation, acceleration, and validation, we ensure robust participation in governance through our mainnet validation for top projects and our involvement in testnets for emerging infrastructures. Our commitment to excellence and transparency makes us a reliable partner to delegate your voting power, helping to secure and innovate within the Celo network."
+  } 
+  "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E": {
+    "name": "Sov",
+    "address": "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E",
+    "logoUri": "/logos/delegatees/"Sov Signal.jpg",
```
Comment (aaronmgdr) [inline src/config/delegates.json:17]:
```suggestion
-    "logoUri": "/logos/delegatees/"Sov Signal.jpg",
+   "logoUri": "/logos/delegatees/SovSignal.jpg",
```

File: src/config/delegates.json [hunk 2]
```diff
@@ -10,6 +10,18 @@
     },
     "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
     "description": "Stake Capital is a trusted and experienced blockchain investment firm, actively contributing to the blockchain ecosystem since 2016. With expertise in venture capital, incubation, acceleration, and validation, we ensure robust participation in governance through our mainnet validation for top projects and our involvement in testnets for emerging infrastructures. Our commitment to excellence and transparency makes us a reliable partner to delegate your voting power, helping to secure and innovate within the Celo network."
+  }
```
Comment (aaronmgdr) [inline src/config/delegates.json:13]:
```suggestion
  },
```

File: src/config/delegates.json [hunk 3]
```diff
@@ -23,6 +23,18 @@
...
+  "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E": {
+    "name": "Sov",
+    "address": "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E",
+   "logoUri": "/logos/delegatees/SovSignal.jpg",
+    "date": "2024-08-31",
+    "links": {
+      "website": "https://forum.celo.org/u/sov/summary",
+      "twitter": "https://x.com/SovereignSignal;"
+    },
+    "interests": ["Ecosystem Development", "Strategic Partnerships", "Grants and Incentives Programs"],
```
Comment (aaronmgdr) [inline src/config/delegates.json:35]:
```suggestion
    "interests": [
      "Ecosystem Development",
      "Strategic Partnerships",
      "Grants and Incentives Programs"
    ],
```

File: src/config/delegates.json [hunk 4]
```diff
@@ -23,6 +23,18 @@
     "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
     "description": "Stake Capital is a trusted and experienced blockchain investment firm, actively contributing to the blockchain ecosystem since 2016. With expertise in venture capital, incubation, acceleration, and validation, we ensure robust participation in governance through our mainnet validation for top projects and our involvement in testnets for emerging infrastructures. Our commitment to excellence and transparency makes us a reliable partner to delegate your voting power, helping to secure and innovate within the Celo network."
   },
+  "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E": {
+    "name": "Sov",
+    "address": "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E",
+   "logoUri": "/logos/delegatees/SovSignal.jpg",
```
Comment (aaronmgdr) [inline src/config/delegates.json:29]:
```suggestion
    "logoUri": "/logos/delegatees/SovSignal.jpg",
```

File: src/config/delegates.json [hunk 5]
```diff
@@ -23,6 +23,22 @@
     "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
     "description": "Stake Capital is a trusted and experienced blockchain investment firm, actively contributing to the blockchain ecosystem since 2016. With expertise in venture capital, incubation, acceleration, and validation, we ensure robust participation in governance through our mainnet validation for top projects and our involvement in testnets for emerging infrastructures. Our commitment to excellence and transparency makes us a reliable partner to delegate your voting power, helping to secure and innovate within the Celo network."
   },
+  "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E": {
+    "name": "Sov",
+    "address": "0x2D7d6Ec6198ADFD5850D00BD601958F6E316b05E",
+    "logoUri": "/logos/delegatees/SovSignal.jpg",
```
Comment (aaronmgdr) [inline src/config/delegates.json]:
```suggestion
    "logoUri": "/logos/delegatees/SovSignal.jpeg",
```

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**